internal/static/internal/css/header.css

```diff
@@ -116,7 +116,7 @@
 
 .dropdown-content a:hover {
     background-color: #103242;
-    color: #BBD8E4;
+    color: rgb(228, 203, 187);
```

Comment:
Why the change here?

Reply:
![Screenshot from 2021-01-11 13-25-18](https://user-images.githubusercontent.com/59800246/104182433-b8b4d200-5410-11eb-9ade-9de65d154a88.png)

I can't recall why